### PR TITLE
chore: upgrade gatsby-plugin-sharp

### LINKED ISSUE #32: failing unit tests
sharp compilation fails on Travis CI due to the Node.js version probably.

### PR DESCRIPTION
**Pull request by DanielRuf**
This also fixes #32

### CODE REVIEW

**Comment by DSchau** [comment]
Deploy preview for *gatsby-blog-starter-kit* ready!

Built with commit 5eb38337a55a4d81ab2fed81e85b5b8cc46ed193

https://deploy-preview-33--gatsby-blog-starter-kit.netlify.com

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+10.6.0
```
Comment (DSchau) [inline .node-version]: Is this something picked up by Travis?

Comment (DSchau) [reply .node-version]: I'd prefer to just use .travis.yml, if so.

**Review by DSchau — APPROVED** [review]
Quick comments, otherwise looks great.  Thank you!

File: .node-version (hunk 1 above)
Comment (DanielRuf) [reply .node-version]: Ah sorry, was my nodenv setting. Wait a minute, I'll remove it.

Comment (DanielRuf) [reply .node-version]: Done.

**Pull request merged**